package.json

```diff
@@ -17,9 +17,12 @@
     "url": "https://github.com/orgs/plantemoran-ai-engineering/.github/issues"
   },
   "scripts": {
-    "lint": "markdownlint **/*.md"
+    "format": "prettier . --write --log-level log",
+    "format:check": "prettier . --check",
+    "lint": "markdownlint **/*.md",
```

Comment:
I think the comma at the end is causing a problem.
Be sure to run all of your scripts locally to ensure your package.json is good
(npm install, lint, format:check)

Reply:
I had accepted your suggestion to pull out linting on markdown, but I didn't pull the project back down and run all the scripts.